### ISSUE TITLE
tsv file

### ISSUE THREAD

**Issue by mowangmodi** [rover-xingyu/Ha-NeRF#10]
There are only 6 tsv files. If I want to train files other than these 6 tsv files, what should I do? Do you want to manually create a tsv file?

**Comment by rover-xingyu** [comment]
Yes,  the tsv files for train/test splits are only provided for [Brandenburg Gate](https://nerf-w.github.io/data/selected_images/brandenburg.tsv), [Sacre Coeur](https://nerf-w.github.io/data/selected_images/sacre.tsv), [Trevi Fountain](https://nerf-w.github.io/data/selected_images/trevi.tsv), [Hagia Sophia](https://nerf-w.github.io/data/selected_images/hagia.tsv), [Prague Old Town](https://nerf-w.github.io/data/selected_images/prague.tsv), and [Taj Mahal](https://nerf-w.github.io/data/selected_images/taj_mahal.tsv), if you want to train on other scenes, you need to clean the data (Section C of NeRF-W) and split the data by yourself.

**Comment by marcostyv** [comment]
> There are only 6 tsv files. If I want to train files other than these 6 tsv files, what should I do? Do you want to manually create a tsv file?

Did you create the tsv files manually? If so, would it be possible to share it in the chat? Otherwise I will try to create my own ones. Thanks.

**Comment by rover-xingyu** [comment]
It is provided by https://nerf-w.github.io/

**Comment by marcostyv** [comment]
> It is provided by https://nerf-w.github.io/

Yes but there are 6. Prague old town and Hagia Sophia datasets do not work due to inconsistences and I do not know why but Taj Majal does not work to me either. Then I only have Brandenburg Gate, Sacre Coeur and Trevi Fountain. 

That is why I ask if the people have other .tsv files such as Temple Nara, Buckingham palace, Colosseum...
